components/SelectorBase/SelectorBase.vue

```diff
@@ -312,15 +311,17 @@ export default {
         const iframeUrl = iframe.contentWindow.location.href;
         // prohibits navigating away from the add form
         if (!iframeUrl.includes(this.popupUrl)) {
-          this.hidePopup();
+          this.hidePopup('');
         }
       } catch (e) {
-        this.hidePopup();
+        this.hidePopup('');
```

Comment:
It should return `null` since `result=null` before the `try`. So if I'm understanding this correctly, we return the payload `null` here thus signifying that the new crop was not found and then handle that in the parent component?

Reply:
Yes. That way it will be handled consistently for the case of `X` and the case where the added item could not be found.